suite-common/wallet-core/src/blockchain/blockchainReducer.ts

```diff
@@ -59,16 +59,21 @@ const connect = (draft: BlockchainState, info: BlockchainInfo) => {
 
     const isHttp = isHttpProtocol(info.url); // can use dynamic backend url settings
 
+    // solana rpc nodes do not have explorer, so we cannot use backend as explorer
+    const isBackendAlsoExplorer = network.networkType !== 'solana';
```

Comment:
I am not very proud of this, but I haven't found any other way to do this. 
The previous implementation was that `if is backed https, then it also is explorer` which is quite inaccurate.

Reply:
imho acceptable